Formula/e/envoy.rb

```diff
@@ -51,21 +62,32 @@ def install
       --verbose_failures
       --action_env=PATH=#{env_path}
       --host_action_env=PATH=#{env_path}
+      --define=wasm=disabled
```

Comment:
is this means wasm was totally disabled?

Reply:
Maybe @mathetake can clarify, but iirc before it wasn't built on Darwin and now (here in this formula) also in linux

Reply:
Correct

Reply:
`wasm=v8` can be restored but Homebrew needs a backport of https://github.com/v8/v8/commit/c814d861f87539c89fc96aed0c6a41ef76f0ff53 (specifically we need `-Wno-cast-function-type-mismatch` for Clang 19) in a Bazel-applied patch file like https://github.com/envoyproxy/envoy/blob/main/bazel/v8.patch

Reply:
we are currently working on updating llvm in envoy - current target is 18.x, but would be happy to review any fixes to make 19+ work

Reply:
> Homebrew needs a backport 

apologies i just read your comment properly

i think we would be less inclined to backport fixes for this to other branches in envoy - simply because there are a bunch of known issues that would need to be addressed to make earlier branches work with llvm > 14

Reply:
Backport for v8 code base.

To be specific, it may be something like:
```diff
diff --git a/bazel/v8.patch b/bazel/v8.patch
index 2d7101db91..32de7740c6 100644
--- a/bazel/v8.patch
+++ b/bazel/v8.patch
@@ -103,3 +103,15 @@ index c3768b8..d4a1dda 100755
  import pdl

  try:
+diff --git a/bazel/defs.bzl b/bazel/defs.bzl
+index e957c0f..a5d142c 100644
+--- a/bazel/defs.bzl
++++ b/bazel/defs.bzl
+@@ -104,6 +104,7 @@ def _default_args():
+                 "-Werror",
+                 "-Wextra",
+                 "-Wno-unknown-warning-option",
++                "-Wno-cast-function-type-mismatch",
+                 "-Wno-bitwise-instead-of-logical",
+                 "-Wno-builtin-assume-aligned-alignment",
+                 "-Wno-unused-parameter",
```

Reply:
for those following along, when you discuss possibility of restoring with some backport, etc. work, you are referring to restoring linux only, or also macOS?